.pubnub.yml

```diff
@@ -0,0 +1,115 @@
+name: kotlin
+version: 1.0.0
```

Comment:
Version number should be `1.0.1`.

Reply:
The version is fine. This is not 1.0.1 yet.